opengemini_client/database_test.py

```diff
@@ -0,0 +1,51 @@
+import time
+import unittest
+from datetime import timedelta
+
+from opengemini_client import client_impl
+from opengemini_client import models
+
+
+class WriteTest(unittest.TestCase):
+
+    def test_create_database(self):
+        cfg = models.Config(address=[models.Address(host='127.0.0.1', port=8086)],
+                            auth_config=models.AuthConfig(auth_type=models.AuthType(0)),
+                            batch_config=models.BatchConfig(batch_size=10, batch_interval=10),
+                            timeout=timedelta(seconds=10), connection_timeout=timedelta(seconds=10),
+                            gzip_enabled=False, tls_enabled=False
+                            )
+        with client_impl.OpenGeminiDBClient(config=cfg) as cli:
+            cli.create_database('create_db_test')
+            res = cli.show_databases()
+            self.assertEqual(True, res.__contains__('create_db_test'))
+
+    def test_create_database_with_rp(self):
+        cfg = models.Config(address=[models.Address(host='127.0.0.1', port=8086)],
+                            auth_config=models.AuthConfig(auth_type=models.AuthType(0)),
+                            batch_config=models.BatchConfig(batch_size=10, batch_interval=10),
+                            timeout=timedelta(seconds=10), connection_timeout=timedelta(seconds=10),
+                            gzip_enabled=False, tls_enabled=False
+                            )
+        with client_impl.OpenGeminiDBClient(config=cfg) as cli:
+            cli.create_database_with_rp('create_db_with_rp_test', models.RpConfig(name='db_with_rp_test',
+                                                                                  duration='1h',
+                                                                                  shard_group_duration='10m',
+                                                                                  index_duration='10m'))
+            qr = cli.query(models.Query(database='create_db_with_rp_test', command='SHOW RETENTION POLICIES',
+                                        retention_policy=''))
+            print(qr)
+
+    def test_drop_database(self):
+        cfg = models.Config(address=[models.Address(host='127.0.0.1', port=8086)],
```

Comment:
You can extract cfg variables into private variables of a class to avoid duplicate code.

Reply:
aggreed